war/src/main/webapp/css/style.css

```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- * 
+ *
```

Comment:
yeah sorry, the editor strips spaces at the line end. they shouldn't be there anyway, so you may see it as improvement :D